### PR TITLE
Callback query and inline keyboards support

### PR DESCRIPTION
**Pull request by MarcoBuster**
I'm adding callback query and inline keyboards support, according to #66.

### CODE REVIEW

File: botogram/objects/callback_query.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
+"""
+    botogram.objects.callback_query
+    Representation of the callback query-related upstream API objects
+
+    Copyright (c) 2015 Pietro Albini <pietro@pietroalbini.io>
```
Comment (pietroalbini) [inline botogram/objects/callback_query.py]: That should be you, not me, and we're not in 2015.

File: botogram/objects/updates.py [hunk 2]
```diff
@@ -24,7 +25,8 @@ class Update(BaseObject):
         "message": Message,
         "edited_message": Message,
         "channel_post": Message,
-        "edited_channel_post": Message
+        "edited_channel_post": Message,
+        "callback_query": CallbackQuery
```
Comment (pietroalbini) [inline botogram/objects/updates.py]: Add a trailing comma here so you don't need to change two lines the next time you want to add a value :)

File: docs/api/telegram.rst [hunk 3]
```diff
@@ -2077,6 +2078,38 @@ about its business.
       *This attribute can be None if it's not provided by Telegram.*
 
 
+.. py:class:: botogram.CallbackQuery
+
+      This class represents a callback query object.
+   .. py:attribute:: id
+
+      Unique identifier for the query
```
Comment (pietroalbini) [inline docs/api/telegram.rst]: *of the callback

File: docs/api/telegram.rst [hunk 4]
```diff
@@ -2077,6 +2078,38 @@ about its business.
...
+
+   .. py:attribute:: sender
+
+      The :py:class:`~botogram.User` that sends the callback query
```
Comment (pietroalbini) [inline docs/api/telegram.rst]: *that sent

File: docs/api/telegram.rst [hunk 5]
```diff
@@ -2077,6 +2078,38 @@ about its business.
...
+
+   .. py:attribute:: message
+
+      :py:class:`~botogram.Message` with the callback button that originated the query.
```
Comment (pietroalbini) [inline docs/api/telegram.rst]: Don't add dots at the end only in some snippets, add them everywhere or nowhere.

File: botogram/objects/callback_query.py (hunk 1 above)
Comment (MarcoBuster) [inline botogram/objects/callback_query.py]: But it's your work :1st_place_medal:

Comment (pietroalbini) [inline botogram/objects/callback_query.py]: I did nothing on that file: the copyright notice should be of the single file, not of the whole project, and you're the author of that file :)

File: docs/api/telegram.rst [hunk 6]
```diff
@@ -2077,6 +2078,47 @@ about its business.
       *This attribute can be None if it's not provided by Telegram.*
 
 
+.. py:class:: botogram.CallbackQuery
+
+   This class represents a callback query received by the bot.
+
```
Comment (pietroalbini) [inline docs/api/telegram.rst]: You forgot the version this is added to.

File: botogram/components.py [hunk 7]
```diff
@@ -171,11 +172,19 @@ def add_channel_post_hook(self, func):
     def add_channel_post_edited_hook(self, func):
         """Add an edited channel post hook"""
         if not callable(func):
-            raise ValueError("A edited channel post hook must be callable")
+            raise ValueError("An edited channel post hook must be callable")
 
         hook = hooks.EditedChannelPostHook(func, self)
         self.__channel_post_edited_hooks.append(hook)
 
+    def add_callback_query_hook(self, func):
```
Comment (pietroalbini) [inline botogram/components.py]: Maybe rename this to `add_callback_hook` to be consistent with `@bot.callback`?

**Comment by pietroalbini** [comment]
I'm implementing a more advanced version of this by myself. Thanks for sending the pull request though!

**Pull request closed without merging**